tt_completion

```diff
@@ -38,41 +41,41 @@ tt_complete() {
 		case "$cmd" in
 		tt|"")
 			completion_base $word ;;
-		manage)
-			completion_manage $word ;;
-		create)
+		create|browse)
 			completion_all_services $word ;;
-		destroy|recreate|ip|browse)
+		destroy|recreate|ip)
 			completion_running_services $word ;;
 		pull|clone|check)
 			completion_services $word ;;
+		workspace)
+			completion_workspace $word ;;
```

Comment:
just keeping the signature consistent, the implementation of the function might make use of it